src/Pages/DetailPage/components/PhotoList.js

```diff
@@ -0,0 +1,40 @@
+import styled from 'styled-components';
+
+function PhotoList({ image }) {
+  return (
+    <PhotoWrapper>
+      <MainPhoto src={image[0].url} />
+      <SubPhotoWrapper>
+        <SubPhoto src={image[1].url} />
+        <SubPhoto src={image[2].url} />
+        <SubPhoto src={image[3].url} />
+        <SubPhoto src={image[4].url} />
+      </SubPhotoWrapper>
+    </PhotoWrapper>
+  );
```

Comment:
```suggestion
function PhotoList({ image }) {
  const [mainImg, ...subImgs] = image;

  return (
    <PhotoWrapper>
      <MainPhoto src={mainImg.url} />
      <SubPhotoWrapper>
        {subImgs.map(({ url }) => <SubPhoto key={url} src={url} />)}
      </SubPhotoWrapper>
    </PhotoWrapper>
  );
```
구조분해 할당과 나머지 연산자를 잘 사용한다면 위와 같이 사용할 수도 있습니다. 직접 배열 만들어서 `const [mainImg, ...subImgs] = image;` 부분이 어떻게 출력되는지 확인해보시면 좋겠습니다.